code/modules/admin/admin_verbs.dm

```diff
@@ -1,3 +1,29 @@
+/datum/admins/proc/admin_sql()
+	set category = "Admin"
+	set name = "SQLQuery"
+	set desc = "Debug Query yo."
+
+	if(!check_rights(R_ADMIN))
+		to_chat(usr, "<span class='warning'>Not enough rights.</span>")
+		return
+	if(!CONFIG_GET(flag/sql_enabled))
+		to_chat(usr, "<span class='warning'>SQL disabled.</span>")
+		return
+	if(!SSdbcore.Connect())
+		to_chat(usr, "<span class='warning'>SQL not connected?!.</span>")
+		return
+	
+	var/rawQuery = input(usr, "query?", "query?", "") as text|null
+	if(!dd_hasprefix(rawQuery, "SELECT"))
```

Comment:
Make it only work with the feedback table (which is not trivial, almost impossible) but I don't see it passing by Jordie/MSO otherwise.

Reply:
I see we have regex so ill make sure it captures it correctly.

Reply:
https://regex101.com/r/7Fa8qE/2/